test/leSpec.js

```diff
@@ -310,53 +321,155 @@ describe('catch all option', function () {
 
 describe('destroys log streams', function () {
     it('default', function () {
-        LE.init(TOKEN);
-        LE.destroy();
+        IOPS.init({
+                token: TOKEN,
+                region: 'eu'
+        });
+        IOPS.destroy();
 
         expect(function () {
-            LE.init(TOKEN);
+            IOPS.init({
+                token: TOKEN,
+                region: 'eu'
+            });
         }).not.toThrow();
     });
 
     it('custom name', function () {
-        LE.init({
+        IOPS.init({
             token: TOKEN,
-            name: 'test'
+            name: 'test',
+            region: 'eu'
         });
-        LE.destroy('test');
+        IOPS.destroy('test');
 
         expect(function () {
-            LE.init({
+            IOPS.init({
                 token: TOKEN,
-                name: 'test'
+                name: 'test',
+                region: 'eu'
             });
         }).not.toThrow();
-        LE.destroy('test');
+        IOPS.destroy('test');
     });
 
     afterEach(destroy);
 });
 
+describe('tests for SSL', function () {
+    beforeEach(mockXMLHttpRequests);
+    beforeEach(addGetJson);
+
+    it('SSL option set to true leads to \"https\"', function () {
+        IOPS.init({
+            token: TOKEN,
+            name: 'test',
+            ssl: true,
+            region: 'eu'
+        });
+        IOPS.log("Test");
+        var url = this.requestList[0].url;
+        assert(url.search('https') === 0);
```

Comment:
`indexOf` would work fine here. No difference in this case, but worth being away that the parameter you pass to `string.search()` is always converted to a regex, so IMHO we shouldn't use it in unless we're actually doing a regex match